src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+    private final NameOfFigure name = NameOfFigure.CIRCLE;
+
+    public Circle(Color color, int radius) {
+        this.setColor(color);
```

Comment:
```suggestion
        super(color);
```

let's use this construct instead - we now call Figure constructor and pass color into it
same in other figures